### DISCUSSION TITLE
article/vercel-domain

### DISCUSSION THREAD

**Discussion by giscus** [tangly1024/tangly1024#33] [Announcements]
# article/vercel-domain

Vercel 借助cloudFlare可以快速绑定解析自己的域名

https://tangly1024.com/article/vercel-domain

**Comment by lanhuakai** [comment]
请问一下：我按流程走完。在我自己的notion blog里增删的内容怎么同步到网站呢？

**Reply by tangly1024** [reply]
增删的内容会自动同步，网页有缓存，多刷新几次即可生效

**Comment by Aiden-QianYu** [comment]
大大好，问一下配置完成后提示：Good news! Your DNS records are set up correctly, but it can take some time for them to propagate globally.  
网页打不开，是需要等嘛？

**Reply by tangly1024** [reply]
是的，【好消息，您的 DNS 记录设置正确，但它们可能需要一些时间才能在全球范围内传播】

**Reply by kakaxi8000** [reply]
你好，哥们，你的网站显示正常了吗？

**Comment by kakaxi8000** [comment]
你好，哪个你说的哪个地址，好像注册不了，搜索域名的适合，老提示选择TLD，下拉框出现的都是白色的，也没有选择的，没法查询域名和注册

**Comment by kakaxi8000** [comment]
大神，根据你所说的绑定了，解析VER里面都正常了
现在浏览器打开提示这个
怎么解决。。。
此页面不能正确地重定向

连接到 www.cgzyss.com 时发生错误。

    有时候禁用或拒绝接受 Cookie 会导致此问题。

**Comment by kakaxi8000** [comment]
等2天吧，不行就换域名解析商了。。。现在还访问不了

**Comment by LxAnC** [comment]
连接已重置怎么办呀，大佬们，总是打不开网页
